tests/queries/0_stateless/02888_system_tables_with_inaccsessible_table_function.sql

```diff
@@ -0,0 +1,34 @@
+-- Tags: no-fasttest
```

Comment:
As I understood table function disabled in the Fasttest. In the first commit CI failed because of:
```
 executeQuery: Code: 46. DB::Exception: Unknown table function postgresql. (UNKNOWN_FUNCTION)
```
https://s3.amazonaws.com/clickhouse-test-reports/55540/15956a690dd4d2217e8dde383fc40a308630ba04/fast_test.html